.github/workflows/ci.yaml

```diff
@@ -30,7 +30,9 @@ jobs:
       - name: Put all tools to store to be cached afterwards
         run: |
           # shellcheck disable=SC2046
-          nix-store -v --realize $( nix-instantiate default.nix )
+          nix-store -v --realize $( nix-instantiate default.nix \
+            -A style -A withTools -A memory -A tests -A release \
+          )
```

Comment:
> Instead of the custom step via nix-store --realize, wouldn't it be enough to add all tools we use in the other jobs to the `tools: ` argument of `actions/setup-nix`?

Yep, good call, let me try

Reply:
> So either add the cache there - or drop it here.

Since there is a separate cache warm-up step, single nix cache per OS, and no job does actually update the cache, the `cache-id` parameter could be safely dropped altogether. Thank you for the heads-up.

Reply:
> Yep, good call, let me try

Tried, works, thank you for the suggestion.